File: src/test/java/testee/it/reportng/ImageToBase64Test.java
package testee.it.reportng;

import org.junit.Ignore;

import static testee.it.reportng.ImageToBase64.base64;
import static testee.it.reportng.ImageToBase64.decodeToImage;
import static testee.it.reportng.ImageToBase64.encodeToString;

public class ImageToBase64Test {
    @Ignore
    public void testImageToBase64() {
        assert encodeToString(decodeToImage(base64), "png").equals(base64) : "Wrong base64 encoding";
    }
}

File: src/test/kotlin/testee/it/reportng/ImageToBase64Test.kt
package testee.it.reportng

import org.junit.Ignore
import org.junit.Test

class ImageToBase64Test {

    @Ignore
    @Test
    fun testImageToBase64() {
        assert(
            ImageToBase64.encodeToString(
                ImageToBase64.decodeToImage(ImageToBase64.base64),
                "png"
            ) == ImageToBase64.base64
        ) { "Wrong base64 encoding" }
    }
}